docs/docusaurus/docs/core/connect_to_data/sql_data/_create_a_batch_definition/_create_a_batch_definition.md

```diff
@@ -0,0 +1,78 @@
+import TabItem from '@theme/TabItem';
+import Tabs from '@theme/Tabs';
+import GxData from '../../../_core_components/_data.jsx'
+import PreReqDataContext from '../../../_core_components/prerequisites/_preconfigured_data_context.md'
+
+### Prerequisites
+- <PreReqDataContext/>.  The variable `context` is used for your Data Context in the following example code.
+- [A Data Asset on a SQL Data Source](#create-a-data-asset).
+
+<Tabs>
+
+<TabItem value="procedure" label="Procedure">
+
+1. Retrieve your Data Asset.
+
+   Replace the value of `datasource_name` with the name of your Data Source and the value of `asset_name` with the name of your Data Asset in the following code.  Then execute it to retrieve an existing Data Source and Data Asset from your Data Context:
+
+   ```python title="Python" name="docs/docusaurus/docs/core/connect_to_data/sql_data/_create_a_data_asset/create_a_data_asset.py retrieve a Data Asset"
+   ```
+
+2. Add a Batch Definition to the Data Asset.
+
+   <Tabs queryString="batch_definition" groupId="batch_definition" defaultValue='full_table' values={[{label: 'Full table', value:'full_table'}, {label: 'Partitioned', value:'partitioned'}]}>
+
+   <TabItem value="full_table" label="Full table">
```

Comment:
I don't know how other folks feel, but I don't view full table batch definitions as being categorically different than the other types of batch definitions. Do they warrant a separate tab?

Reply:
I generally give a separate explanation for anything that has different implementation details.  In this case, the parameters for Full Table Batch Definitions are not the same as the parameters for Query Batch Definitions, so even though you can do the same things with them once they've been created I try to explain the creation process individually.